airflow/providers/redis/provider.yaml

```
@@ -35,11 +35,12 @@ versions:
 
 dependencies:
   - apache-airflow>=2.4.0
-  # Redis 4 introduced a number of changes that likely need testing including mixins in redis commands
-  # as well as unquoting URLS with `urllib.parse.unquote`:
-  # https://github.com/redis/redis-py/blob/master/CHANGES
-  # TODO: upgrade to support redis package >=4
-  - redis~=3.2
+  # Upgrading redis package >= 4 doesn't require code changes
+  # Changes in Redis 4 only brings into impact when we are directly importing redis.client
+  # to execute the commands, which we are not doing.
+  # The issue with unquoting URLS with `urllib.parse.unquote` is also not applicable
+  # to airflow, because we are not using the from_url function
```

```suggestion
  # Upgrading redis package >= 4 doesn't require code changes
  # Changes in Redis 4 only brings into impact when we are directly importing redis.client
  # to execute the commands, which we are not doing.
  # The issue with unquoting URLS with `urllib.parse.unquote` is also not applicable
  # to airflow, because we are not using the from_url function
```

I think what @eladkal ment (and Also what I think) - you can remove the whole comment now and simply make  it part of commit message. There is no point in leaving in the code, but the commit message will carry the trace of this being verified.

---

yeah sorry for not being clear.
If version is OK no need to further explain why it's OK.
We explain only things that we need to consider for future work

---

Thanks all. I removed the comments and make it as part of commit msg. I'm fixing the static check error and the build-doc error that I get from the CI tests. Will let you guys know once they are fixed.